R/README.md

```diff
@@ -0,0 +1,260 @@
+|10     |sale                     |3                      |6800.55  |3     |Sharon   |2             |
+
+Suppose you would like to do some analysis such as sale price by make of car or
+find the most popular color for each model. To do so, you can
+[_join_](https://en.wikipedia.org/wiki/Join_%28SQL%29) the tables together.
+After inner joining the *primary* table to the *sale* table on `car_id` and
+selecting only the fields you are interested in, your resulting analytic dataset
+might look something like this:
+
+| car_id|make     |model    |price    |color |customer |
+|------:|:--------|:--------|:--------|:-----|:--------|
+|      1|AMC      |Javelin  |12000.50 |1     |Bob      |
+|      1|AMC      |Javelin  |13750.77 |3     |Sue      |
+|      1|AMC      |Javelin  |15004.57 |2     |Kim      |
+|      3|Camaro   |Z28      |7800.00  |2     |Janice   |
+|      3|Camaro   |Z28      |8000.00  |3     |Tim      |
+|      4|Chrysler |Imperial |7500.00  |1     |Jim      |
+|      7|Duster   |360      |8756.40  |4     |Sarah    |
+|      7|Duster   |360      |6800.88  |2     |Pablo    |
+|      7|Duster   |360      |8888.88  |1     |Erica    |
+|      7|Duster   |360      |970.00   |4     |Juan     |
+|     10|Merc     |230      |7800.98  |2     |Ted      |
+|     10|Merc     |230      |7954.00  |1     |Quentin  |
+|     10|Merc     |230      |6800.55  |3     |Sharon   |
+
+Such a join can be accomplished numerous ways. Just to name a few:
+
+- [`dplyr::*_join`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.5/topics/join)
+- [`sqldf::sqldf`](https://www.rdocumentation.org/packages/sqldf/versions/0.4-11/topics/sqldf)
+- [`base::merge`](https://www.rdocumentation.org/packages/base/versions/3.5.0/topics/merge)
+
+## Instructions
+
+### Installation
+
+First you must install the package. To do so, execute the following in your R console:
+
+```r
+install.packages("REDCapRITS")
+```
+
+To install the development version, run the following code:
+
+```r
+if (!requireNamespace(remotes, quietly = TRUE)) install.packages("remotes")
+remotes::install_github("SpectrumHealthResearch/REDCapRITS/R")
+```
+
+### Usage
+
+After the package is installed, follow these instructions:
+
+1. Download the record dataset and metadata (data dictionary). This can
+be accomplished by several methods:
+    - Using the API. Check with your REDCap administrator for details.
+    - Exporting the data from the web interface by selecting *CSV / Microsoft Excel (raw data)*.
+    - Exporting the data from the web interface by selecting *R Statistical Software*.
+      If you use this method, you may run the R script supplied by REDCap prior to splitting the data.
+    - **Do NOT** export from the web interface with the *CSV / Microsoft Excel (labels)* option.
+      This will not work with `REDCapRITS`.
+1. Call the function, pointing it to your record dataset and metadata
```

Comment:
Good observation! In this case, I chose to use `1.` for each item on purpose because the Markdown language allows us to use any [arbitrary numbers for enumerated lists](https://www.markdownguide.org/basic-syntax/#ordered-lists). This is helpful because we may want to re-arrange items during the editing process.